rabbitmq/json-exchanges.yml

```diff
@@ -0,0 +1,123 @@
+type: json
+data:
+  url: /api/exchanges
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: exchanges
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: exchanges.message_stats.ack
+        source: message_stats/ack
+        type: long_gauge
```

Comment:
Is it really a gauge, not a counter?

Reply:
It is a gauge

Reply:
What exactly does it show? Count of acknowledged messages ... "in last minute", "in last 5 minutes", "in last hour", "since the last restart"...?

Reply:
@nmtien ping? See my previous question?

Reply:
updated to counter